app/src/main/java/io/github/droidkaigi/confsched/api/DroidKaigiClient.java

```diff
@@ -21,6 +21,7 @@
 
     private static final String END_POINT = "https://raw.githubusercontent.com";
     private static final String JSON_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+    private static final String SESSIONS_API_ROUTES = "/konifar/droidkaigi2016/ar_translations/app/src/main/res/raw/";
```

Comment:
I'll ix this before merge.
From `/konifar/droidkaigi2016/ar_translations/app/src/main/res/raw/`
To `/konifar/droidkaigi2016/master/app/src/main/res/raw/`